README.md

```diff
@@ -10,57 +10,33 @@ Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- HTML-CSS
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-### Setup
+-Flexbox
+-Linters
+-Figma
+-GitHub Flow
 
 ### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+git clone git@github.com:Tobinchilongo/Portfolio-Setup.git
+cd Portfolio-Setup
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 Tobin Chilongo Jnr
 
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/Tobinchilongo)
+- Twitter: [@twitterhandle](https://twitter.com/Tobin_Official)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tobin-chilongo-a6736415a/)
```

Comment:
[Optional] Kindly replace the `@githubhandle, @twitterhandle, and LinkedIn` with your personal handles. This will make your README look more professional (@githubhandle, @